ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.ts

```diff
@@ -62,7 +62,7 @@ export class AnalysisContextFormComponent extends FormComponent
         },
         {
             "id": 100,
-            "name": "Migration to Red Hat JBoss EAP 6",
+            "name": "Migration to JBoss EAP 6",
             "source": null,
```

Comment:
I didn't do `None -> Cloud readiness`, because it was awaiting Marc's feedback. But he already replied, so I changed that.